dapps-profiling/src/profilings/tokens/index.ts

```diff
@@ -1,167 +1,165 @@
 import { Network } from "stellar-plus/lib/stellar-plus/types";
+import { createBaseAccounts, setupAssets } from "./setup";
+
 import {
-  addTrustlinesToUsers,
-  createBaseAccounts,
   mintSorobanTokensToUsers,
-  setupAssets,
-} from "./setup";
+  setupDemuUsers,
+} from "../../utils/simulation/functions";
+import { DemoUser } from "../../utils/simulation-types";
+import {
+  profileBurn,
+  profileMinting,
+  profilePayments,
+} from "./profiling-simulations";
 import { exportArrayToCSV } from "../../utils/export-to-csv";
+import { Console } from "console";
 
 export type tokensProfilingConfigType = {
   nUsers: number;
   nTransactions: number;
   network: Network;
+  transactions?: tokenTransactions[];
 };
 
+export enum tokenTransactions {
+  transfer = "transfer",
+  // transferFrom = "transferFrom",
+  // approve = "approve",
+  mint = "mint",
+  burn = "burn",
+}
+
+/**
+ * @args {tokensProfilingConfigType} args
+ * @param {number} args.nUsers - Number of users to be created
+ * @param {number} args.nTransactions - Number of transactions to be executed in each testing phase
+ * @param {Network} args.network - Network to be used
+ * @param {tokenTransactions[]} args.transactions - Transactions to be executed. If not provided, all transactions will be executed
```

Comment:
With destructuring you can do this:
```suggestion
 * @args {tokensProfilingConfigType} args
 * @param {number} nUsers - Number of users to be created
 * @param {number} nTransactions - Number of transactions to be executed in each testing phase
 * @param {Network} network - Network to be used
 * @param {tokenTransactions[]} transactions - Transactions to be executed. If not provided, all transactions will be executed
```

Reply:
Adjusted